tools/gc_decode.cpp

```diff
@@ -44,10 +48,21 @@ int main(int argc, char *argv[]) {
     argv_offset++;
   }
 
+  if (strncmp("-gc", argv[argv_offset], 3) == 0) {
+    dumpgc = true;
```

Comment:
Bad variable name. You are not dumping GlobalCache, you're saying the "input" is global cache.